lib/evmone/eof.hpp

```diff
@@ -91,7 +91,7 @@ enum class EOFValidationError
     invalid_max_stack_height,
     no_terminating_instruction,
     stack_height_mismatch,
-    non_empty_stack_on_terminating_instruction,
+    code_section_outputs_mismatch_on_retf,
```

Comment:
It is also used for JUMPF in #644 
```suggestion
    code_section_outputs_mismatch,
```